ledger/participant-state/kvutils/src/main/scala/com/daml/ledger/participant/state/kvutils/InputsAndEffects.scala

```diff
@@ -88,9 +88,7 @@ private[kvutils] object InputsAndEffects {
                 GlobalKey(create.coinst.template, forceNoContractIds(keyWithMaintainers.key.value)))
             }
 
-          case exe @ NodeExercises(_, _, _, _, _, _, _, _, _, _, _, _, _, _) =>
-            addContractInput(exe.targetCoid)
-
```

Comment:
I liked this blank line.

Reply:
The auto-formater drop the one under `case NodeExercises(_, _, _, _, _, _, _, _, _, _, _, _, _, _) =>`
I drop the other for the sake of consistency.

Reply:
I added back the blank lines together with a `()` to trick the auto-formater